response_operations_ui/templates/conversation-view.html

```diff
@@ -0,0 +1,43 @@
+{% extends "layouts/base.html" %}
+
+{% set subject = messages[-1]["subject"] or "No Subject" %}
+
+{% block page_title %} {{ subject }} | Survey Data Collection {% endblock %}
+
+{% block main %}
+
+
+<h1 class="saturn" name="page-messages-title">{{ subject }}</h1>
+
+<div class="secure-message-conversation block">
+    <div class="secure-message-sent-message-meta">
+         <div class="secure-message--meta venus"><span class="secure-message-conversation-meta__label">Survey: </span><span id="survey">{{ messages[0].get('survey') }}</span></div>
```

Comment:
🎨 I would probably do it like `messages[0]['survey']` unless we're looking for a default value

Reply:
If you assign `messages[0]` to a variable
```
{% set first_message = messages[0] %}
```
Then you should be able to use `{{ first_message.survey }}`